models/orderItem.go

```diff
@@ -10,7 +10,7 @@ type OrderItem struct {
 	MealID       uint
 	PriceE5      int64
 	Quantity     int64
-	PaidQuantity int64       `gorm:"default:false"`
+	PaidQuantity int64       `gorm:"default:0"`
```

Comment:
what's your intention here? @phella

Reply:
it was boolean at the beginning.

switched to int in case a user paid only 1 meal not the full ordered quantity.